kernel/api/src/main/java/org/sakaiproject/messaging/api/MicrosoftMessage.java

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
Hey, please update the years! 2003-2017 not good :D

Reply:
Done!